src/widgets/stack_sidebar.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013-2015, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+//! A controller for StackSidebar
+
+use ffi;
+use cast::{GTK_STACK_SIDEBAR, GTK_STACK};
+use FFIWidget;
+
+/// An automatic sidebar widget
+struct_Widget!(StackSidebar);
+
+impl StackSidebar {
+    pub fn new() -> Option<StackSidebar> {
+        let tmp_pointer = unsafe { ffi::gtk_stack_sidebar_new() };
+        check_pointer!(tmp_pointer, StackSidebar)
+    }
+
+    pub fn set_stack(&self, stack: ::Stack) {
+        unsafe {
+            ffi::gtk_stack_sidebar_set_stack(GTK_STACK_SIDEBAR(self.pointer),
+                                             GTK_STACK(stack.unwrap_widget()))
+        }
+    }
+
+    pub fn get_stack(&self) -> Option<::Stack> {
+        let tmp_pointer = unsafe {
+            ffi::gtk_stack_sidebar_get_stack(GTK_STACK_SIDEBAR(self.pointer))
+        };
+
+        if tmp_pointer.is_null() {
+            None
+        } else {
+            Some(::FFIWidget::wrap_widget(tmp_pointer as *mut _))
+        }
```

Comment:
It seems this bit could be handled with `check_pointer!` like in `new` and cut down on the verbosity.